crates/bevy_input/src/keyboard.rs

```diff
@@ -1,16 +1,31 @@
 use crate::{ButtonState, Input};
-use bevy_ecs::event::EventReader;
-use bevy_ecs::system::ResMut;
+use bevy_ecs::{event::EventReader, system::ResMut};
 
-/// A key input event from a keyboard device
+/// A keyboard input event.
+///
+/// This event is the translated version of the `WindowEvent::KeyboardInput` from the `winit` crate.
+/// It is available to the end user and can be used for game logic.
+///
+/// ## Usage
+///
+/// The event is read inside of the [`keyboard_input_system`](crate::keyboard::keyboard_input_system)
```

Comment:
Should this say that the event is _updated_ by the system? It's a bit confusing to me to only say it's read by a system.

Reply:
Right, it does get cleared by the system so _consumed_ does sounds better in that case.